tests/common.py

```diff
@@ -26,6 +26,8 @@
 from timemachine.utils import path_to_internal_file
 
 HILBERT_GRID_DIM = 128
+# Directory to write files to that will be stored as artifacts in CI.
+ARTIFACT_DIR_NAME = "pytest-artifacts"
```

Comment:
Is there any benefit to this being an env var? (e.g. so it could be read by both the CI configuration and the pytest process?)